cypresse2e/cypress/integration/e2eTests/privacyPolicy/privacyPolicy.spec.js

```diff
@@ -0,0 +1,261 @@
+import PrivatePolicyPage from "../../../elements/pages/privacyPolicy";
```

Comment:
the imported class name should match the class name exported from the POM file

Reply:
Yes, that is correct too.  @jessica-lau